.github/workflows/build.yaml

```
@@ -38,21 +38,15 @@ jobs:
     steps:
       - name: Remove label
         if: github.event_name == 'pull_request'
-        uses: actions/github-script@v7
+        uses: actions/github-script@v4
```

actionlint (1.7.4)

the runner of "actions/github-script@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)